### DISCUSSION TITLE
Welcome to XC=BASIC Discussions!

### DISCUSSION THREAD

**Discussion by neilsf** [neilsf/XC-BASIC#130] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gpowerf** [comment]
Hi there, I recently came across your project here. 

I'm planning to slowly start using it over the next few months, I got the compiler working today and did a basic hello world. 

I'll submit feedback when needed :)

**Reply by neilsf** [reply]
Cool. Let me know if you're stuck.... cause it might happen. Being a compiled language it may be very hard to debug. But I think the speed you can gain is worth the struggle :-)
